### ISSUE TITLE
duplicate rewrite when override a class

### ISSUE THREAD

**Issue by matiasserrano** [enriquepiatti/Magicento#192]
If I override a class and then I try to override it again using Magicento (and selecting the same module to create the new class - this could happen if the user forgets where is the override, for example) two things happens:

1) Magicento tries to create again the class (PHPStorm stops this action with an error: "File already exists"
2) Magicento adds again the XML line to the override

This issue could be considered of less impact since is doesn't broke Magento, but the Override Conflicts Checker detects this as a conflict.

**Comment by enriquepiatti** [comment]
Hi @matiasserrano I will try to add a better check for that, but a fast way to check if a class has already a rewrite is to see the icon on the left of the class name, that icon is from PhpStorm and shows you the classes that are extending that class (and normally rewrites are extending the original class)

**Comment by matiasserrano** [comment]
Yes, I knew it... in fact, I always use it. But today I realized while overwriting a very big class (and sincerely, I was so tired that I didn't check the icon...). As I told you, it doesn't be a big issue at all and we can live with that...  I didn't test it but perhaps the same occurs with all the other rewrites.
